micah-akpan/questioner-api
# #163221363 Fix Multiple Votes Bug

#### What does this PR do?
- Fixes the Multi-vote (upvote/downvote) bug
#### Description of Task to be completed?
Fix Multiple votes on questions bug. 
A single cannot upvote a question after he/she has downvoted the question and vice-versa. 
This fix makes it possible for a user to be able to downvote a upvote a question if he or she has downvoted the question and vice versa, without adding or subtracting more than a single vote from the total votes
#### How should this be manually tested?
- Clone project repo
- Install all dependencies: `npm install`
- Add env variables to .env file, based on the template in .env-sample file
- Run server: `npm run dev:start`
- Using Postman, head over to the request bar and enter the URL: localhost:9999/api/v1/questions/1/upvote
- To test the fix:
  - Try upvoting/downvoting on a single more than once
- Refer to the [docs](https://questionerapi.docs.apiary.io)  to see how to create a question
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163221363
#### Screenshots (if appropriate)
#### Questions:
